docs/en/Running-HBase-on-Alluxio.md

```diff
@@ -35,10 +35,22 @@ Change the `hbase.rootdir` property in `conf/hbase-site.xml`:
 ```xml
 <property>
   <name>hbase.rootdir</name>
-  <value>alluxio://<ALLUXIO_MASTER_HOSTNAME>:<PORT>/hbase</value>
+  <value>alluxio://master_hostname:port/hbase</value>
```

Comment:
is this change consistent with other docs?

Reply:
Every file has their own ways. At least the alluxio://<ALLUXIO_MASTER_HOSTNAME>:<PORT> is only used once here, but the alluxio://master_hostname:port is using many times in Hive doc.

The most common use is take a example hostname and port:alluxio://localhost:19998